### ISSUE TITLE
Question about eval configuration

### ISSUE THREAD

**Issue by qianyi1016** [mli0603/openpi-comet#3]
Thank you for the detailed presentation of your solution.
I have observed that your team employed models that were more specifically matched to each task for evaluation. I'm very interested in the underlying configuration for this process.
Could you please provide the specific evaluation configuration you used? Or was a unified standard​ applied across all 50 tasks?

**Comment by DelinQu** [comment]
Hi [qianyi1016](https://github.com/qianyi1016), thank you for your interest and for taking the time to review our solution. 
1. Evaluation Configuration
We actually used a unified evaluation configuration across all 50 BEHAVIOR-1K tasks.
Evaluation script: https://github.com/mli0603/openpi-comet/blob/df7e7e5dae80c971caaa8c3b4fdb1b994ffd793d/scripts/serve_b1k.py#L66

The key parameters are:
```bash
control_mode: "receeding_horizon"
max_len: 32
```

2. Model Selection
Due to time constraints during the challenge, we ultimately submitted the model that performed best on each public validation set. We are now training a fully pretrained unified model that can handle all sub-tasks without task-specific variations. Its performance has already reached Q-score > 0.30. 

I will push an update tonight to simplify the codebase, unify some implementations, and double-check all evaluation parameters to make them clearer. Thank you again for raising this issue — greatly appreciated! 🤗

**Comment by qianyi1016** [comment]
> Did you use the Full Resolution `RgbWrapper` https://github.com/mli0603/openpi-comet/blob/main/data_generation/rollout/learning/wrappers/rgb_wrapper.py?你使用的是 Full Resolution `RgbWrapper` https://github.com/mli0603/openpi-comet/blob/main/data_generation/rollout/learning/wrappers/rgb_wrapper.py 吗？
> 
@DelinQu No, I didn't use  `Full Resolution RgbWrapper`, I didn't find how to use it in Evaluation stage (it seems appear in 'RFT post training' and Run the 'RFT rollout in parallel') Could you show me how to use it? 

As I haven't tried RFT so I didn't copy the `openpi_comet/data_generation/rollout/learning` to `BEHAVIOR-1K/OmniGibson/omnigibson/learning`. Do you mean I also need to do it?

Than you again for your time and patience.

**Comment by qianyi1016** [comment]
@DelinQu Hi, Thank you for your reply (I accidentally closed the issue, sorry, but this problem is not resolved yet). I changed this as soon as possible but the success rate is still 0. It seems there is still some details that I didn't align. For example, in issue [4](https://github.com/mli0603/openpi-comet/issues/5), I saw that you used different fine_grained_level for the evaluation of each task. I tried using one change like fine_grained_level=1, but it seems that merely modifying the parameter is not enough. Without modifying the parameter, I evaluated 4 tasks, namely task 15, task 16, as well as task 30 and task 32, and their success rates were very poor at 0. I would like to know if there is anywhere else I haven't aligned? The configuration I used is:
```
1. receding horizon
2. max_len = 32
3. RGBWrapper
4. The corresponding model in the model zoo：
30,32[openpi_comet/pi05-b1kpt12-cs32](https://huggingface.co/sunshk/comet_weights/blob/main/pi05-pt12-pretrain-40k.zip)
15[openpi_comet/pi05-b1kpt50-cs32](https://huggingface.co/sunshk/comet_weights/blob/main/pi05-pt50-pretrain-40k.zip)
16[pi05-pt10-pretrain-40k](https://huggingface.co/sunshk/comet_weights/blob/main/pi05-pt10-pretrain-40k.zip)
5. fine_grained_level = 0
```
I know this is a bit troublesome. If you are willing, could you update the more detailed configuration for each task's evaluation in your future work? Thank you again for your work.

**Comment by DelinQu** [comment]
Hi @qianyi1016 , feel free to reopen this issue.

This is our submission model accuracy (not Q score), corresponding to the Model Zoo per task checkpoints. As you can see, the scores for 32: wash_a_baseball_cap, 30: setting_the_fire, 15: bringing_in_wood, 16: moving_boxes_to_storage are not very high, close to 20%, which is why everyone's average score is less than 30%.

```bash
bringing_in_wood                                   score 2
bringing_water                                     score 9
cook_hot_dogs                                      score 10
freeze_pies                                        score 1
hanging_pictures                                   score 3
hiding_Easter_eggs                                 score 3
make_microwave_popcorn                             score 10
moving_boxes_to_storage                            score 10
outfit_a_basic_toolbox                             score 1
picking_up_trash                                   score 8
preparing_lunch_box                                score 2
putting_away_Halloween_decorations                 score 3
putting_dishes_away_after_cleaning                 score 1
putting_shoes_on_rack                              score 4
rearranging_kitchen_furniture                      score 1
set_up_a_coffee_station_in_your_kitchen            score 1
setting_the_fire                                   score 1
slicing_vegetables                                 score 1
spraying_for_bugs                                  score 1
spraying_fruit_trees                               score 3
turning_on_radio                                   score 10
wash_a_baseball_cap                                score 4
🤗 Total ACC: 89
```

The corresponding Q scores are as follows. Note that this is different from the leaderboard results because we didn't have time to test all public validation sets. The complete scores might be higher.

```json
{
   "team": "comet",
   "affiliation": "",
   "date": "2025-11-17",
   "track": "standard",
   "overall_scores": {
       "q_score": 0.2223507936507936,
       "time_score": 0.29005980282293603,
       "base_distance_score": 0.5824459975003197,
       "left_distance_score": 0.11605023766154744,
       "right_distance_score": 0.12516607012167286
   }
}
```

The unified pretrained model we're building is already showing good performance. Please stay tuned, and thank you!

```json

{
   "team": "comet",
   "affiliation": "",
   "date": "2025-12-02",
   "track": "standard",
   "overall_scores": {
       "q_score": 0.3131380952380952,
       "time_score": 0.33521952706639374,
       "base_distance_score": 0.6955934809461772,
       "left_distance_score": 0.1253475926321848,
       "right_distance_score": 0.13272875460847663
   }
}

```

**Comment by qianyi1016** [comment]
@DelinQu Hi, I really appreciate your help with my questions. Your work and initiatives are truly meaningful, and they have been a great help to me. I will continue to follow your project and look forward to your future publications. 🤗
